src/Features/CSharp/Portable/EditAndContinue/SyntaxComparer.cs

```diff
@@ -195,6 +196,7 @@ private static int TiedToAncestor(Label label)
             case Label.Parameter:
             case Label.AttributeList:
             case Label.Attribute:
+            case Label.ExtensionBlockDeclaration:
```

Comment:
Do we have a test for this scenario (moving extension block to a new parent)?

Reply:
Added a test.